src/Compilers/Server/VBCSCompiler/IClientConnection.cs

```diff
@@ -41,7 +41,24 @@ internal interface IClientConnection : IDisposable
 
     internal interface IClientConnectionHost
     {
-        Task<IClientConnection> ListenAsync(CancellationToken cancellationToken);
-    }
+        bool IsListening { get; }
```

Comment:
Missing doc comment? Everything else has one.

Reply:
I'm listening to this feedaback 😄